src/ui/map.js

```diff
@@ -3239,8 +3239,8 @@ function removeNode(node) {
  * };
  *
  * HelloWorldControl.prototype.onRemove = function () {
```

Comment:
still using `function` here

Reply:
This is an es5 example and an arrow function wouldn't work because `this` would not be what we want

Reply:
Ah ok, I didn't look closely enough